go/adbc/driver/flightsql/flightsql_statement.go

```diff
@@ -37,11 +37,17 @@ import (
 
 const (
 	OptionStatementQueueSize = "adbc.flight.sql.rpc.queue_size"
+	// Explicitly set substrait version for Flight SQL
+	// substrait *does* include the version in the serialized plan
```

Comment:
it wasn't at the time and people suggested we add it in case; now it's vestigial unfortunately